.github/workflows/release.yml

```diff
@@ -42,7 +42,7 @@ on:
   pull_request:
   push:
     tags:
-      - '**[0-9]+.[0-9]+.[0-9]+*'
+      - 'cargo-near-v[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
@dj8yfo Have you considered using the `tag-namespace` configuration for cargo-dist? https://github.com/axodotdev/cargo-dist/blob/main/book/src/reference/config.md#tag-namespace